### PR TITLE
Add counting lesson

### PR DESCRIPTION
**Pull request by Shedinho20**
I test the function by passing an array which returned the right result. However, the main function keeps returning an error

### CODE REVIEW

**Comment by elishaking** [comment]
fixed the issue

File: src/arrays/counting.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  A.forEach(function(item, index, array) {
+    if (item % 2 == 0) {
+      item = item / 2;
+      if (item % 2 == 0) {
+        item = item / 2;
+        if (item % 2 == 0) {
+          result.push(index);
+        }
+      }
+    }
```
Comment (elishaking) [inline src/arrays/counting.js]: replace this with a while loop

File: src/arrays/counting.js [hunk 2]
```diff
@@ -14,19 +14,20 @@ function solution(A) {
   result = [];
```
Comment (elishaking) [inline src/arrays/counting.js]: always declare your variables with a `let` or `const`. 
```
const result = [];
```

File: src/arrays/counting.js [hunk 3]
```diff
@@ -14,19 +14,20 @@ function solution(A) {
...
   // todo: implement function here
   A.forEach(function(item, index, array) {
-    if (item % 2 == 0) {
-      item = item / 2;
-      if (item % 2 == 0) {
-        item = item / 2;
-        if (item % 2 == 0) {
-          result.push(index);
-        }
+    count = 0;
```
Comment (elishaking) [inline src/arrays/counting.js]: declare the variable with the `let` keyword

Comment (Shedinho20) [reply src/arrays/counting.js]: okay. I will work on it now

**Comment by elishaking** [comment]
I'll add some more exercises to this branch shortly

**Pull request merged**